examples/server/tests/README.md

```diff
@@ -44,4 +44,10 @@ To run with stdout/stderr display in real time (verbose output, but useful for d
 DEBUG=1 ./tests.sh -s -v -x
 ```
 
+Hint: You can compile and run test in single command, useful for local developement:
+
+```shell
+cmake --build build -j --target llama-server && ./examples/server/tests/tests.sh
```

Comment:
@ggerganov FYI, the change in `tests.sh` should allow you to run test script from anywhere, not necessary need to `cd tests`